src/test/rgw/CMakeLists.txt

```diff
@@ -205,3 +205,11 @@ install(TARGETS ceph_test_rgw_gc_log DESTINATION ${CMAKE_INSTALL_BINDIR})
 
 add_ceph_test(test-ceph-diff-sorted.sh
   ${CMAKE_CURRENT_SOURCE_DIR}/test-ceph-diff-sorted.sh)
+
+
+if(WITH_RADOSGW_AWS_ENDPOINT)
+  # unittest_rgw_aws
+  add_executable(unittest_rgw_aws test_rgw_aws.cc)
+  add_ceph_unittest(unittest_rgw_aws)
```

Comment:
we cannot automatically run this UT, because it requires credentials.
so, please remove this line.
the executable will still be created and we can run it manually

Reply:
on removing this line, I am getting `fatal error: gtest/gtest.h: No such file or directory`

Reply:
please see here: https://github.com/ceph/ceph/blob/master/src/test/cls_2pc_queue/CMakeLists.txt
for an example of a unit test which is built but not executed automatically